validator_worker/src/sentry_interface.rs

```diff
@@ -83,7 +82,7 @@ impl<T: Adapter + 'static> SentryApi<T> {
         }))
         .await
         {
-            handle_http_error(e);
+            println!("Propagation error: {}", e);
```

Comment:
let's log the error here

Reply:
Since we don't have the logging setup in `validator_worker` and it's not as straight forward as I though and nothing to do with the dependencies update I would like to do this in separate PR.